### ISSUE TITLE
Accessibility: Narrator reads markdown that is passed through the localize method

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#2001]
## To Reproduce
1. Turn on Narrator
1. Go to full-feature Mock Bot
1. Type 'timestamp' and send
1. Note that Narrator says  message "First message at time t = 0." as: "First message at time underscore t underscore equals 0"

## Expected behavior
Markdown should be stripped from text before being passed to `localize`

## 
Additional context
use the remark plugin called '[strip markdown](https://www.npmjs.com/package/strip-markdown)'

[Bug]
